src/App.tsx

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import { Exercise0 } from './exercises/00-Relay-Compiler/Exercise0';
 import { Exercise1 } from './exercises/01-Query-Renderer/Exercise1';
 import { Exercise2 } from './exercises/02-Fragment-Container/Exercise2';
+import { Exercise3 } from './exercises/03-Testing-Queries/Exercise3';
```

Comment:
We talked while mobbing about how there's a lot of busy work to spin up a new exercise (adding routes, copying & renaming components, etc.). As this project grows, there are lots of opportunities to reduce that friction.